modules/local/arborview.nf

```diff
@@ -4,8 +4,8 @@ process ARBOR_VIEW {
     stageInMode 'copy' // Need to copy in arbor view html
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-    "docker.io/python:3.11.6" :
-    "docker.io/python:3.11.6" }"
+    "biocontainers/python:3.12" :
```

Comment:
Can you change this to `https://depot.galaxyproject.org/singularity/python%3A3.12`?

This means that if the the boolean expression above is true (for singularity) it will download the cached version from depot.galaxyproject.org